src/__test__/components/data-management/ProjectCard.test.jsx

```diff
@@ -49,13 +49,13 @@ describe('ProjectCard', () => {
     expect(screen.getByText(samplesIdsArray.length)).toBeInTheDocument();
 
     // Created date is shown
-    expect(screen.getByText(moment(createdDate).fromNow())).toBeInTheDocument();
+    expect(screen.getAllByText(moment(createdDate).fromNow())[0]).toBeInTheDocument();
```

Comment:
I think this test started failing when I merged from master for some reason, it has nothing to do with all my changes in this branch

Reply:
with this change it seems to have been fixed

Reply:
if I can get more opinions on this please go ahead